tests/spec/core/xref-spec.js

```diff
@@ -502,7 +497,10 @@ describe("Core — xref", () => {
         </section>
       </section>
     `;
-    const config = { xref: { url: xrefTestUrl("refs") }, localBiblio };
+    const config = {
+      xref: { url: `${location.origin}/tests/data/xref/refs.json` },
```

Comment:
keeping this one for now, as the file contains fake dataset (due to lack of exported informative references in dataset)